examples/node-groups/managed-node-groups/main.tf

```diff
@@ -134,6 +134,10 @@ module "eks_blueprints" {
       eni_delete        = true
       public_ip         = false # Use this to enable public IP for EC2 instances; only for public subnets used in launch templates
 
+      http_endpoint               = "enabled"
+      http_tokens                 = "optional"
+      http_put_response_hop_limit = 3
```

Comment:
Should we change this in another example or create a new one? We probably want the default MNG example to use the default options.

Reply:
I believe [this](https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/657/files/698fea88e7db215cb116aeddfa676ad51edea4d4#diff-a74a95b7fe2f941d719e545e427114d9c5b80d1131125d8d2d698b35249004ebR107) is the "default" or minimum config node group. I added to this node group since it looked like it was demonstrating all the options available, but I am happy to move to a different node group example